lib/go/Makefile

```diff
@@ -15,15 +15,34 @@ endif
 export GOPATH
 
 
+################################################################################
+##                                   DEP                                      ##
+################################################################################
+
+DEP ?= ./dep
+DEP_VER ?= 0.3.1
+DEP_BIN := dep-$$GOHOSTOS-$$GOHOSTARCH
+DEP_URL := https://github.com/golang/dep/releases/download/v$(DEP_VER)/$$DEP_BIN
+
+$(DEP):
```

Comment:
I don't spend much time designing makefiles, but a little iffy in automatically installing a binary for people. We can add how to get `dep` in the README or in a contributing guide.